docs/config_entries_index.md

```diff
@@ -132,19 +44,23 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
 
 ## Unloading entries
 
-Components can optionally support unloading a config entry. When unloading an entry, the component needs to clean up all entities, unsubscribe any event listener and close all connections. To implement this, add `async_unload_entry(hass, entry)` to your component ([example](https://github.com/home-assistant/core/blob/f18ddb628c3574bc82e21563d9ba901bd75bc8b5/homeassistant/components/hassio/__init__.py#L534)).
+Integrations can optionally support unloading a config entry. When unloading an entry, the integration needs to clean up all entities, unsubscribe any event listener and close all connections. To implement this, add `async_unload_entry(hass, entry)` to your integration ([example](https://github.com/home-assistant/core/blob/f18ddb628c3574bc82e21563d9ba901bd75bc8b5/homeassistant/components/hassio/__init__.py#L534)). The state of the config entry is set to `ConfigEntryState.UNLOAD_IN_PROGRESS` before `async_unload_entry` is called.
 
 For each platform that you forwarded the config entry to, you will need to forward the unloading too.
 
 ```python
-await self.hass.config_entries.async_forward_entry_unload(self.config_entry, "light")
+async def async_unload_entry(hass: HomeAssistant, entry: MyConfigEntry) -> bool:
+    """Unload a config entry."""
+    if (unload_ok := await hass.config_entries.async_unload_platforms(entry, PLATFORMS)):
+        entry.runtime_data.listener()
```

Comment:
```suggestion
    unload_ok := await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
```
can we make the example simpler?

Reply:
Why, I think it's good with a somewhat realistic example?

Reply:
I guess there should be comments though to make it clearer what's going on. FWIW the example is copied from the quality scale rule about implementing unload support.

Reply:
I think it makes it look like you we need to call `entry.runtime_data.listener()` as part of "forward the unloading too".
For someone new, it makes it harder to understand what is actually the code we want to exemplify vs. extras. Maybe a comment would help making it clear that its just an extra example.

Reply:
I cleared the example so the body of the function is just a docstring